app/commands/git/sync_practice_exercise.rb

```diff
@@ -31,8 +31,7 @@ def exercise_config_modified?
       return false unless track_config_modified?
 
       exercise_config[:slug] != exercise.slug ||
-        # TODO: enable the line underneath when (if?) practice exercises have names
-        # exercise_config[:name] != exercise.title ||
+        exercise_config[:name] != exercise.title ||
```

Comment:
This will break for any tracks prior for which the v3 migration PR hasn't been merged

Reply:
OK, cool :+1: